app/views/reservations/index.html.erb

```diff
@@ -7,6 +7,7 @@
     <%=resa.start_date%>
     <%=resa.end_date%>
     <%=resa.total_price%>
+    <%= link_to 'Payer', new_reservation_payment_path(resa) %>
```

Comment:
Cool ! Now, we can implment th method to display this link, only when the reservation is accepted.